paymentsheet/src/main/java/com/stripe/android/paymentsheet/addresselement/AutocompleteScreen.kt

```diff
@@ -93,10 +94,10 @@ internal fun AutocompleteScreenUI(viewModel: AutocompleteViewModel) {
             }
         },
         bottomBar = {
-            val background = if (isSystemInDarkTheme()) {
-                MaterialTheme.stripeColors.component
+            val background = if (MaterialTheme.stripeColors.materialColors.surface.shouldUseDarkDynamicColor()) {
+                Color.Black.copy(alpha = 0.07f)
             } else {
-                MaterialTheme.stripeColors.materialColors.surface.darken(0.07f)
+                Color.White.copy(alpha = 0.07f)
```

Comment:
Think when we synced on this, the plan was to use `0.2f`, did this change?

Reply:
Based on [this thread](https://stripe.slack.com/archives/CFA2HJ99A/p1716320732743449) we decided to go with 7%. Archived thread in ticket